src/App.css

```diff
@@ -1,38 +1,40 @@
-.App {
-  text-align: center;
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.item-container {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 10px;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.checkbox-item {
+  -webkit-transform: scale(2);
+  margin: 8px;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+
+.item-texto {
+  height: 45px;
+  margin-right: 10px;
+  margin-left: 10px;
   display: flex;
-  flex-direction: column;
+  justify-content: flex-start;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  font-size: 30px;
 }
 
-.App-link {
-  color: #61dafb;
-}
+.btn-remover-item {
+  all: unset;
```

Comment:
what does this do? never used this property before

Reply:
Restores all the properties of an element.
In this case, I use it to remove the button's properties.

Reply:
![Alt Text](https://media.giphy.com/media/V0IdVIIW1y5d6/giphy.gif)